ddl/db_test.go

```diff
@@ -1792,6 +1792,7 @@ func (s *testDBSuite) TestCreateTableWithHashPartition(c *C) {
 	s.tk = testkit.NewTestKit(c, s.store)
 	s.tk.MustExec("use test;")
 	s.tk.MustExec("drop table if exists employees;")
+	s.tk.MustExec("set @@session.tidb_enable_table_partition = 1")
```

Comment:
Remember to have been removed before？

Reply:
Test range partition doesn't not need to set this variable, because it's enabled by default.
It's removed at that time 

Test hash partition has to set it.